lib/sequenceserver/blast/job.rb

```diff
@@ -50,6 +60,11 @@ def command
                      " -query '#{qfile}' #{options}"
       end
 
+      def database_ncharacter_length
+        total_database_length_arr = databases.map(&:ncharacters)
```

Comment:
would calling this database_lengths be more appropriate @iQuxLE ?

Reply:
Sure, I'll add that.

Reply:
It is not clear if database_length refers to number of sequences in the database or number of characters.
- I think `ncharacters_total` might be a more appropriate name for your method.
- I don't think you need a variable within the method.


For context, we borrow as much of blast's terminology as possible. `ncharacters` is obtained from `blastdbcmd`'s output and it probably uses the same name.